src/plugins/tabs/components/tabs.vue

```diff
@@ -49,11 +52,12 @@
     <div
         v-for="tab in tabsList"
         :key="tab.keyString"
+        :style="getTabStyles(tab)"
```

Comment:
There are too many method calls from the template in the tabs view.

Let's do this by computed property. Define a computed property that calculates styles for all tabs and returns them as a map (it will only do it when it needs to). Key it on tab.keyString.